tools/functions.sh

```diff
@@ -125,3 +127,21 @@ do_curl_tarball_with_path() {
     tar -C "${dldir}/${cmd}" -xvf "${dldir}/${cmd}.tar.gz"
     mv "${dldir}/${cmd}/${url_and_path[1]}" "${path}/${cmd}"
 }
+
+do_curl_txt() {
+    local cmd="${1}"
+    local url="${2}"
+    local default_path="${HOME}/.wego/bin"
```

Comment:
Not using the default_path variable so it can be removed.

Reply:
I am using it to set the path. Like the other functions are doing.